firefly/app.py

```diff
@@ -22,6 +23,10 @@
 ctx = threading.local()
 ctx.request = None
 
+loader = FileSystemLoader(searchpath=get_template_path())
+env = Environment(loader=loader)
+template = env.get_template('index.html')
```

Comment:
You can use `PackageLoader` instead computing the template path. See <http://jinja.pocoo.org/docs/2.10/api/#basics>.

Reply:
Ah! I didnt know about this. This is a robust abstraction.